Python.md

```diff
@@ -40,7 +85,37 @@ result = connection.execute(sqlalchemy.text("""
            show_gravatar
       FROM "user"
      WHERE musicbrainz_id IN :musicbrainz_usernames
-"""), {
-    'musicbrainz_usernames': tuple(usernames),
-})
+""")
+result = connection.execute(query, {'musicbrainz_usernames': tuple(usernames)})
 ```
+
+It is preferred to assign the query to a separate variable and not include it inside the
+`execute` call.
```

Comment:
That's why `SELECT` in the existing example goes on the new line. Take a look at it again.
Same with arrays or dicts. That's why I write
```python
random(function([
    "foo",
    "bar",
]
```
and not
```python
random(function(["foo",
                 "bar"]
```